src/shared/ui/select/Select.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Button, Menu } from '@material-ui/core';
+
+import { Checkbox, Field } from '..';
+
+import csx from './Select.scss';
+
+export interface SelectItem {
+  label: string;
+  value: boolean;
+}
+
+export interface SelectProps {
+  label: string;
+  items: SelectItem[];
+  placeholder?: string;
+  onSelect(event: React.ChangeEvent<HTMLInputElement>, checked: boolean): void;
+}
+
```

Comment:
Just a question: why are you creating these interfaces in the component instead of model.ts?

Reply:
This is mistake. It will be moved to model.ts file in next PR